composer.json

```diff
@@ -19,7 +19,8 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "container-interop/container-interop": "^1.2",
+        "container-interop/container-interop": "^1.2.0",
```

Comment:
We don't need above line. We are not using this library here anymore.
We need of course `conflict` entry, as it is now below.